src/cascadia/TerminalApp/ActionPreviewHandlers.cpp

```diff
@@ -146,7 +146,8 @@ namespace winrt::TerminalApp::implementation
         const auto backup = _restorePreviewFuncs.empty();
 
         _ApplyToActiveControls([&](const auto& control) {
-            control.PreviewInput(args.Input());
+            const auto& str{ args.Input() };
+            control.PreviewInput(str);

```

Comment:
Is this an accidental change?

Reply:
It was a debugging change. I'll revert that